pages/status.tsx

```diff
@@ -136,6 +137,11 @@ const Status: FC = () => {
     [formik]
   )
 
+  //When the page is conditionally rendered it maintains its spot on the page. Calling this will force it to scroll to the top of the page.
+  const scrollToTop = () => {
```

Comment:
Use `useEffect` hook instead.

```
useEffect(() => {
  window.scrollTo(0, 0)
}, [checkStatusResponse])
```

Page will scroll to top on component mount and when his dependency `checkStatusResponse` change.

Reply:
https://stackoverflow.com/a/39345165

Reply:
I think this is the way I had it originally but without the dependency. Settled on the function instead but this is much cleaner, thanks.